console/console-init/ui/src/modules/connection-detail/components/ConnectionDetailHeader/ConnectionDetailHeader.tsx

```diff
@@ -66,28 +66,28 @@ export const ConnectionDetailHeader: React.FunctionComponent<IConnectionHeaderDe
   return (
     <Card>
       <CardTitle>
-        <Title id="cd-header-title" headingLevel="h1" size="4xl">
+        <Title id="connection-detail-header-title" headingLevel="h1" size="4xl">
           {hostname}
         </Title>
       </CardTitle>
       <CardBody>
         <Flex>
           <FlexItem
-            id="cd-header-container-id"
+            id="connection-detail-header-container-id-flexitem"
```

Comment:
Please avoid using ids like 'flexitem'. Instead you can use the functionality which this element is referring to for eg, section or input etc.

Reply:
It was discussed that Flex items does not really need an ID. I did not make any changes to the files I had already worked on, but I don't think changing flex item's id would be of any help. What do you think?